qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -286,7 +286,10 @@ def compose(self, other, qargs=None, front=False):
             z4[:, qargs] = z3
             pauli_list = PauliList(BasePauli(z4, x4, phase))
 
-        coeffs = np.kron(self.coeffs, other.coeffs)
+        # note: the following is a faster code equivalent to
+        # `coeffs = np.kron(self.coeffs, other.coeffs)`
+        # since `self.coeffs` and `other.coeffs` are both 1d arrays.
```

Comment:
I'm not sure whether I need to keep this note. Any thought?

Reply:
It's possibly not necessary, but it also doesn't hurt.  I'd just leave it.